metricbeat/tests/system/test_http.py

```diff
@@ -33,12 +41,15 @@ def test_json(self):
         self.assertEqual(len(output), 1)
         evt = output[0]
 
+        print(json.dumps(evt))
+
         assert evt["http"]["test"]["hello"] == "world"
 
         # Delete dynamic namespace part for fields comparison
         del evt["http"]["test"]
 
         self.assertItemsEqual(self.de_dot(HTTP_FIELDS), evt.keys(), evt)
+        print(json.dumps(evt))
```

Comment:
Leftover? Did you plan to add here something around the new header you specified above?